src/axom/inlet/Inlet.cpp

```diff
@@ -179,5 +179,15 @@ bool Inlet::get(const std::string& name, std::string& value)
   return true;
 }
 
+void Inlet::registerDocWriter(std::shared_ptr<DocWriter> writer) {
+  m_docWriter = writer;
+}
+
+void Inlet::writeDocs() {
+  if (m_docWriterEnabled) {
```

Comment:
Do you really need a variable to track if docs are enabled?  
Or would it be sufficient to check that `m_docWriter != nulltpr` ?

In other words, would you even have a case where `m_docWriter` is set, but `m_docWriterEnabled == false`?